### PR TITLE
Moon Activity fix 

### PR DESCRIPTION
**Pull request by ricknjacky**
Corrected my previous mistakes,in previous PR
the activity is working fine, though the popup doesn't load up

### CODE REVIEW

**Comment by ricknjacky** [comment]
@prakashujjwal1010 I have done the changes you suggested, please review and tell me where can I correct myself further, as the pop up doesn't load up

**Comment by prakashujjwal1010** [comment]
include `libnotify.css` in `index.html` also

File: activities/Moon.activity/data.ini [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [*]
+MoonImageSaved=Image was was saved to journal!
```
Comment (prakashujjwal1010) [inline activities/Moon.activity/data.ini:2]: 2 times 'was'. make it one :)

Comment (ricknjacky) [reply activities/Moon.activity/data.ini:2]: I will do that :) in a minute please review

**Comment by ricknjacky** [comment]
Thanks a ton @prakashujjwal1010 and @llaske ! for being so patient with my mistakes throughout, it's working fine now.

**Comment by ricknjacky** [comment]
#867 resolved!

**Comment by llaske** [comment]
Good work @ricknjacky. Thanks.

**Pull request merged**